scripts/tests/test-ci-all.sh

```diff
@@ -221,7 +221,7 @@ echo "$parsed_test_commands" | if parallel \
   --halt-on-error 1 \
   --joblog "$joblog" \
   --timeout 600 \
-  --load 150% \
+  --load 1000% \
```

Comment:
@dpc wants to see our CPUs burn :laughing:

![Q4tDgE7](https://github.com/fedimint/fedimint/assets/84478054/4d09d4e4-93a7-48b3-ae39-bfdb61776bf1)

Reply:
Let it rip!

I played with tweaking this in the previous refactor (https://github.com/fedimint/fedimint/pull/4413) and noticed flakiness with the latency tests. Worth keeping an eye on.

- Should we also relax `--nice 15`?
- Would it make sense to make `--load` profile dependent? Aggressive in CI but slower locally?

Reply:
On modern systems `nice` doesn't really do much (it's relative only to the current terminal session, or something like that, and not global), so tweaking nice doesn't really change much one way or the other.

If I run it locally I want it to be done ASAP as well. I was thinking about making it `10000` an locally I tested 30000.

If anything gets flaky that's great, maybe we've uncovered a bug. Unless it's about something getting too slow, then I guess we'll back off a bit. :D

Reply:
One Linux you want to be using `chrt` and `ionice`. Here is a cargo alias I use locally:

```
exec ionice -c 3 nice -n 20 chrt -i 0 cargo "$@"
```

and on Nix, you want to set your builder scheduling cvlasses to:

```
  nix = {
    extraOptions = ''
      experimental-features = nix-command flakes
    '';
    settings = {
      max-jobs = "auto";
      auto-optimise-store = true;
    };
    daemonIOSchedClass = "idle";
    daemonIOSchedPriority = 7;
    daemonCPUSchedPolicy = "idle";
  };
```